addOns/sstiscanner/src/main/java/org/zaproxy/zap/extension/sstiscanner/DjangoTemplateFormat.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.sstiscanner;
```

Comment:
Should use the new package.

Reply:
org.zap.addon.sstiscanner

Reply:
Missing `zaproxy`.

Reply:
I'm now wondering if it needs to be in its own add-on, some parts are being moved and in the end it will not be that big anymore.

Reply:
> Missing zaproxy.

Updated.

Reply:
I moved the blind rule to ascanrules alpha, soon will move the other one. Should I create a folder to add all the classes except the scan rule?

Reply:
That sounds good.

Reply:
@thc202 were you thinking move the entire thing to ascanrulesAlpha?

Reply:
Yes.